pkg/crypto/tinkcrypto/testdata/sender_1pu_ref.hex

```diff
@@ -0,0 +1 @@
+6caf13723d14850ad4b42cd6dde935bffd2fff00a9ba70de05c203a5e1722ca7
```

Comment:
can any of these one line files be combined into some JSON files?

Reply:
it will require a new type to unmarshal too.. it's feasible

Reply:
done